{{cookiecutter.project_slug}}/ansible/production_inventory

```diff
@@ -1,7 +1,14 @@
 [web]
-todo.your.host.name
+todo.your.production.host.name
 
 [web:vars]
-site_name={{ cookiecutter.project_slug }}.lizard.net
-checkout_name="master"
-buildout_cfg=production.cfg
+site_name = {{ site_name }}
+project_slug = {{ project_slug }}
+package_name = {{ package_name }}
+project_path: "/srv/{% raw %}{{ site_name }}{% endraw %}"
+django_settings = {{ package_name }}.settings
+supervisor_port = todo.put.port.here
+gunicorn_port = todo.put.port.here
+gunicorn_workers = 3
+gunicorn_max_requests = 500
```

Comment:
Dat is niet iets dat we normaliter instellen, dus het hoeft niet zo prominent bij de instellingen te staan.

(Misschien later nog eens kijken of deze waarde wel optimaal is, überhaupt).

Reply:
Je bedoelt, gewoon de gunicorn defaults aanhouden?

Reply:
Het is al 7 jaar geleden dat ik het oorspronkelijk heb ingesteld... ik dacht dat de default was om *niet* te herstarten na een bepaald aantal requests.

Dat het ingesteld was kwam doordat er bij het lezen van shapefiles soms memory leaks waren. Daar ben je dan tenminste na een tijdje gegarandeerd vanaf.

=> je kan het laten staan, maar misschien is het niet meer nodig. Of is het juist wel heel hard nodig!